code/modules/mob/living/carbon/human/species/station/skeleton.dm

```diff
@@ -0,0 +1,63 @@
+/datum/species/skeleton
+	name = SPECIES_SKELETON
+	name_plural = "skeletal remains"
+	blurb = "They look like human remains. Some poor soul expired here, a million miles from home."
+
+	icobase = 'icons/mob/human_races/r_skeleton.dmi'
+	deform = 'icons/mob/human_races/r_skeleton.dmi'
+	damage_overlays = null
+	damage_mask = null
+	blood_mask = null
+
+	min_age = 499	// Oldest skeleton is of a person of 140 years. Implies OS managed to extend life expectancy via technology. Revise according to lore.
+	max_age = 359	// OS disappeared in 2291, CEV Eris launched 2642. This means the skeleton of a child of 8 years would be 359 years old.
```

Comment:
Are skeletons One Star exclusive?

Reply:
The things I need these for aren't related to onestar, they are still going to see a lot of use in onestar areas but its nice to have some flexibility

Reply:
Alrighty. Skeletons are theme-agnostic and age can be set with the spawner.